src/styles/main.scss

```diff
@@ -1,3 +1,26 @@
-body {
-  margin: 0;
+@import "./blocks/body.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/header-moyo";
+@import "./blocks/button.scss";
+@import "./blocks/card.scss";
+@import "./blocks/price.scss";
+@import "./blocks/card__rating.scss";
+@import "./blocks/after.scss";
+@import "./blocks/variables.scss";
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+  border-radius: 5px;
+  padding-top: 50px;
+  max-width: $number-of-cards *
+    $card-width + ($number-of-cards - 1) * $card-gap;
+  margin: 0 auto;
+}
+
+.catalog :nth-child(4n) {
+  margin-right: 0;
```

Comment:
Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

Reply:
corrected